api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/WorksParams.scala

```diff
@@ -142,12 +146,65 @@ object MultipleWorksParams extends QueryParamsUtils {
         "items.locations.accessConditions.status".as[AccessStatusFilter].?,
         "type".as[WorkTypeFilter].?,
         "partOf".as[PartOfFilter].?,
-        "_queryType".as[SearchQueryType].?,
-        "_index".as[String].?,
       )
-    ).tflatMap { args =>
-      val params = (MultipleWorksParams.apply _).tupled(args)
-      validated(params.paginationErrors, params)
+    ).tflatMap {
+      case (
+          page,
+          pageSize,
+          format,
+          dateFrom,
+          dateTo,
+          languages,
+          genre,
+          subjects,
+          contributors,
+          license,
+          includes,
+          aggregations,
+          sort,
+          sortOrder,
+          query,
+          identifiers,
+          itemLocationType,
+          itemLocationTypeId,
+          accessStatus,
+          workType,
+          partOf) =>
+        // Scala has a max tuple size of 22 so this is nested to get around this limit
```

Comment:
lol, I noticed we were about to hit this the other day. Possibly worth a refactor into `parameterMap` at some point

Reply:
fwiw the tuple unpacking is slightly nicer in the latest version of akka-http which I've upgraded to in this PR https://github.com/wellcomecollection/catalogue/pull/1230 that I need to get round to merging and applying